examples/json_map/merge_copied.mapping.json

```diff
@@ -0,0 +1,35 @@
+{
+    "/@default": "entry",
+    "/ENTRY[entry]/DATA[data]/current": "/entry/data/current",
+    "/ENTRY[entry]/DATA[data]/current_295C": "/entry/data/current_295C",
+    "/ENTRY[entry]/DATA[data]/current_300C": "/entry/data/current_300C",
+    "/ENTRY[entry]/DATA[data]/current_305C": "/entry/data/current_305C",
+    "/ENTRY[entry]/DATA[data]/current_310C": "/entry/data/current_310C",
+    "/ENTRY[entry]/DATA[data]/temperature": "/entry/data/temperature",
+    "/ENTRY[entry]/DATA[data]/voltage": "/entry/data/voltage",
+    "/ENTRY[entry]/INSTRUMENT[instrument]/ENVIRONMENT[environment]/voltage_controller/calibration_time": "/entry/instrument/environment/voltage_controller/calibration_time",
+    "/ENTRY[entry]/INSTRUMENT[instrument]/ENVIRONMENT[environment]/voltage_controller/run_control": "/entry/instrument/environment/voltage_controller/run_control",
+    "/ENTRY[entry]/INSTRUMENT[instrument]/ENVIRONMENT[environment]/voltage_controller/value": "/entry/instrument/environment/voltage_controller/value",
```

Comment:
Would it be possible to add group copying?
This would mean writing
```json
"/ENTRY[entry]/INSTRUMENT[instrument]/ENVIRONMENT[environment]/voltage_controller": "/entry/instrument/environment/voltage_controller",
```
here. Or did we just wanted to add group copying later?

Reply:
I would also find the linking syntax here helpful. So we always specify which file this comes from (or at least are able to do it) for cases where we might have duplicate fields.

This is what I'm thinking of:
```json
"/ENTRY[entry]/INSTRUMENT[instrument]/ENVIRONMENT[environment]/voltage_controller/run_control": 
"voltage_and_temperature.nxs:/entry/instrument/environment/voltage_controller/run_control",
```

Reply:
ok, sounds good